[2025-03-13T21:22:56.166+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-13T21:22:56.173+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:54.781244+00:00 [queued]>
[2025-03-13T21:22:56.177+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:54.781244+00:00 [queued]>
[2025-03-13T21:22:56.180+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-13T21:22:56.191+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform> on 2025-03-13 21:22:54.781244+00:00
[2025-03-13T21:22:56.200+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=145) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-13T21:22:56.202+0000] {standard_task_runner.py:72} INFO - Started process 146 to run task
[2025-03-13T21:22:56.205+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'transform', 'manual__2025-03-13T21:22:54.781244+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmp8i1i2804']
[2025-03-13T21:22:56.208+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask transform
[2025-03-13T21:22:56.423+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.transform manual__2025-03-13T21:22:54.781244+00:00 [running]> on host 7499a3ac39c6
[2025-03-13T21:22:56.456+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-13T21:22:54.781244+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-13T21:22:54.781244+00:00'
[2025-03-13T21:22:56.457+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-13T21:22:56.465+0000] {logging_mixin.py:190} INFO - Starting transformation process...
[2025-03-13T21:22:56.465+0000] {logging_mixin.py:190} INFO - Data trasnformed
[2025-03-13T21:22:56.466+0000] {logging_mixin.py:190} INFO - Transformation complete.
[2025-03-13T21:22:56.472+0000] {python.py:240} INFO - Done. Returned value was: Transformation completed successfully!
[2025-03-13T21:22:56.482+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-13T21:22:56.483+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=transform, run_id=manual__2025-03-13T21:22:54.781244+00:00, execution_date=20250313T212254, start_date=20250313T212256, end_date=20250313T212256
[2025-03-13T21:22:56.515+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-13T21:22:56.614+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-13T21:22:56.615+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
